mkosi/resources/convert_to_rolling_rhino.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -e
+
+cd $(dirname $0)
+[ -d rhino-convert ] && rm -r rhino-convert
+[ -d convert ] && rm -r convert
+
+#TODO: Uncomment when the below PR is merged:
+#
+#   https://github.com/rollingrhinoremix/rhino-convert/pull/2
+#
+#git clone https://github.com/rollingrhinoremix/rhino-convert.git
+
+#TODO: Remove these lines when the above PR is merged
+git clone https://github.com/mcassaniti/rhino-convert.git
+cd rhino-convert
+git checkout origin/convert-vs-creation-directories-v1 > /dev/null 2>&1
+cd ..
```

Comment:
```suggestion
{
    cd rhino-convert
    git checkout origin/convert-vs-creation-directories-v1 &>/dev/null
}
```

Reply:
subshells are better for quickly going somewhere and popping back. Our shell style has no spaces between redirection operator and filename.